source/wp-content/themes/wporg-developer-2023/patterns/front-page-content.php

```diff
@@ -7,166 +7,166 @@
-<h2 class="wp-block-heading is-style-default has-link-color" style="margin-bottom:var(--wp--preset--spacing--20)"><?php esc_attr_e( 'Documentation', 'wporg' ); ?></h2>
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"},"blockGap":"var:preset|spacing|50"}},"className":"entry-content"} -->
+<div class="wp-block-group alignfull entry-content" style="padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)"><!-- wp:group {"align":"full","style":{"spacing":{"padding":{"right":"var:preset|spacing|edge-space","left":"var:preset|spacing|edge-space"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull" style="padding-right:var(--wp--preset--spacing--edge-space);padding-left:var(--wp--preset--spacing--edge-space)"><!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide"><!-- wp:heading {"style":{"typography":{"fontStyle":"normal","fontWeight":"700","lineHeight":1.6}},"className":"is-style-short-text","fontSize":"normal","fontFamily":"inter"} -->
+<h2 class="wp-block-heading is-style-short-text has-inter-font-family has-normal-font-size" style="font-style:normal;font-weight:700;line-height:1.6"><?php esc_html_e( 'Documentation', 'wporg' ); ?></h2>
 <!-- /wp:heading -->
 
-<!-- wp:paragraph {"style":{"spacing":{"margin":{"top":"0"}}}} -->
-<p style="margin-top:0"><a href="https://developer.wordpress.org/block-editor" data-type="helphub_article"><?php esc_attr_e( 'Block Editor', 'wporg' ); ?></a></p>
-<!-- /wp:paragraph -->
-
-<!-- wp:paragraph {"style":{"spacing":{"margin":{"top":"0"}}}} -->
-<p style="margin-top:0"><a href="https://developer.wordpress.org/themes" data-type="helphub_article"><?php esc_attr_e( 'Themes', 'wporg' ); ?></a></p>
-<!-- /wp:paragraph -->
+<!-- wp:list {"className":"is-style-cards-3-col","fontSize":"small"} -->
+<ul class="is-style-cards-3-col has-small-font-size"><!-- wp:list-item -->
+<li><a href="https://developer.wordpress.org/block-editor"><strong><?php esc_html_e( 'Block Editor', 'wporg' ); ?></strong><br><?php esc_html_e( 'Creating the building blocks of WordPress?', 'wporg' ); ?></a></li>
+<!-- /wp:list-item -->
 
-<!-- wp:paragraph {"style":{"spacing":{"margin":{"top":"0"}}}} -->
-<p style="margin-top:0"><a href="https://developer.wordpress.org/plugins"><?php esc_attr_e( 'Plugins', 'wporg' ); ?></a></p>
-<!-- /wp:paragraph -->
+<!-- wp:list-item -->
+<li><a href="https://developer.wordpress.org/themes"><strong><?php esc_html_e( 'Themes', 'wporg' ); ?></strong><br><?php esc_html_e( 'Want to learn how to start theming WordPress?', 'wporg' ); ?></a></li>
```

Comment:
Should these be `h3`s?

Reply:
Hmm, now that you mention it the treatment does suggest that. Can't do an h3 within a list item in the editor, so would have to change it to a group of groups. Would be really nice to be able to do the grid layout in the editor in that case, need to try to find that setting...

Reply:
Found the experimental flag for grid. It means I can do about half the layout in the editor with groups and h3s, still needs some custom CSS.

Reply:
Now converted to a grid group of h3s. Spacing between bold text and description increased, should be pretty close to Figma.